docs/course_materials/exercises/03_Variables_and_datatypes/homework_variables.md

```diff
@@ -3,12 +3,13 @@
 ## Variables, numbers and boolean 
```

Comment:
done

Reply:
Please add - HomeWork at the end of the line